### PR TITLE
feat: written blog test 📝📝📝

### PR DESCRIPTION
**Pull request by Aju100**
In order to run on your machine , use python manage.py test

### CODE REVIEW

File: blog/tests.py [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+User = get_user_model()
+
+class BlogTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        # create a user
+        testuser = User.objects.create_user(
+            username = 'testuser1',
+            password = '123456789'
+        )
```
Comment (razyesh) [inline blog/tests.py:14]: not username, authenticate user using email, so email is also must field. I have already merged accounts app, go through that

File: blog/tests.py [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+        title = f'{post.title}'
+        subtitle = f'{post.subtitle}'
+        meta_description = f'{post.meta_description}'
+        content = f'{post.content}'
+        
+        self.assertEqual(user,'testuser')
+        self.assertEqual(title,'New Blog title')
+        self.assertEqual(subtitle,'simple title')
+        self.assertEqual(meta_description,' a web app blog testing haha')
+        self.assertEqual(content,'Content will goes here.')
```
Comment (razyesh) [inline blog/tests.py:38]: perform test in API end point

File: blog/tests.py (hunk 1 above)
Comment (Aju100) [reply blog/tests.py:14]: thank u @razyesh

File: blog/tests.py (hunk 2 above)
Comment (Aju100) [reply blog/tests.py:38]: I haven't made API one. It's for model testing. API is on todo list.

Comment (Aju100) [reply blog/tests.py:38]: @razyesh  can you please run this test, I got some error in db.sqlite3.

File: blog/tests.py (hunk 1 above)
Comment (Aju100) [reply blog/tests.py:14]: @razyesh  I will be updating it.

**Comment by Aju100** [comment]
@razyesh  PR should be done right ?

**Pull request merged**